src/watch_filter.rs

```diff
@@ -173,15 +200,54 @@ impl Filterer for WatchFilter {
         }
 
         // By process of elimination the event is valid
-        tracing::trace!("passing event: {event:?}");
+        tracing::debug!("passing event: {event:?}");
         Ok(true)
     }
 }
 
+#[derive(Debug)]
+pub struct Config {
```

Comment:
1000000% banned from calling this `Config`.  We already have 800 things called `Config` and I can no longer keep track of them.  I'd suggest something like `WatchPatterns` or `WhatToWatch` maybe?

Reply:
Err...this is a very common pattern in rust packages...

Reply:
To be more explicit: I think this is an appropriate name and certainly shouldn't be banned. Naming a type `Config` - like `Result`, `Error`, `Iter`, `Entry` - is a well-established convention. They can be differentiated by consumers by renaming (`use std::io::Error as IoError`) or module path (`watch_filter::Config`).

As a blatant appeal to authority, `wasmtime` appears to have 6 `struct Config`s: https://github.com/search?q=repo%3Abytecodealliance%2Fwasmtime+%22struct+Config%22&type=code